Dockerfile.win10.min

```diff
@@ -97,7 +97,7 @@ RUN git clone --single-branch --depth=1 -b %VCPGK_VERSION% https://github.com/mi
 WORKDIR /vcpkg
 RUN bootstrap-vcpkg.bat
 RUN vcpkg.exe update
-RUN vcpkg.exe install openssl:x64-windows openssl-windows:x64-windows rapidjson:x64-windows re2:x64-windows boost-interprocess:x64-windows boost-stacktrace:x64-windows zlib:x64-windows pthread:x64-windows b64:x64-windows
+RUN vcpkg.exe install openssl:x64-windows openssl-windows:x64-windows rapidjson:x64-windows re2:x64-windows boost-interprocess:x64-windows boost-stacktrace:x64-windows boost-core:x64-windows zlib:x64-windows pthread:x64-windows b64:x64-windows
```

Comment:
@nv-kmcgill53 @mc-nv 

I believe I caught an intermediate windows build state that caused this - but that it shouldn't technically be required - as it should be a dependency already 

let me know if we should remove it or keep it -

Reply:
I don't see a problem listing it here.